dev-docs/bidders/twistDigital.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: bidder
+title: Twist Digital
+description: Prebid Twist Digital Bidder Adaptor
+biddercode: twistdigital
+userIds: britepoolId, criteo, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId
+media_types: banner, video
+gvl_id: 1292
+tcfeu_supported: true
+gpp_supported: true
+usp_supported: true
+pbjs: true
+sidebarType: 1
+---
```

Comment:
Can you please update additional fields like

coppa_supported
schain_supported
dchain_supported
safeframes_ok
deals_supported
floors_supported
pbs
multiformat_supported
fpd_supported
...